miri-script/src/main.rs

```diff
@@ -38,21 +38,15 @@ impl Subcommands {
         }
     }
     fn get_auto_config(&self, miri_dir: &Path) -> Option<AutoConfig> {
-        use std::env::VarError;
-        if let Err(VarError::NotPresent) = std::env::var("MIRI_AUTO_OPS") {
```

Comment:
Removing this entirely is not what I said. We do need to check if it is set, and skip the auto-ops if so.

We just don't have to set it ourselves.